templates/institutional-landing/institutional-landing-bad.html

```diff
@@ -0,0 +1,72 @@
+---
+title: "[Institution name]"
+layout: no-container
+language: "en"
+altLangPage: "institutional-landing-fr.html"
```

Comment:
Considering there is no translation for this page and it is only used for testing. Just don't provide an `altLangPage` which will auto remove the language toggle link.

```suggestion
```